### DISCUSSION TITLE
Lenguaje de programacion

### DISCUSSION THREAD

**Discussion by felixrolando** [Schweinepriester/github-profile-achievements#182] [Q&A] [answered]
Luego de leer y ver varios videos de los diferentes lenguajes de programación, me gustaria saber cual de los diferentes lenguaje es mas popular en la actualidad.

**Comment by LJMartinez07** [comment, marked as answer]
**Information by stackoverflow  developer survey:**

1. JavaScript
2. Python
3. Java

**Reply by felixrolando** [reply]
thanks
